src/app/workspace/[workspaceId]/toolbar.tsx

```
@@ -153,14 +162,122 @@ export const WorkspaceToolbar = ({ children }: WorkspaceToolbarProps) => {
+								setAiSearchInput("");
+							}}
+							size="sm"
+							title="Toggle AI Search"
+							variant="ghost"
+						>
+							<Sparkles className="h-4 w-4" />
+						</Button>
+						<div className="flex-1">
+							{useAI ? (
+								<input
+									autoFocus
```

Biome (2.4.4)

[error] Avoid the autoFocus attribute.

(lint/a11y/noAutofocus)